app/components/CredentialCard/CredentialCard.tsx

```diff
@@ -16,14 +16,16 @@ type CredentialCardProps = {
 
 export default function CredentialCard({ rawCredentialRecord }: CredentialCardProps): JSX.Element {
   const { credential } = rawCredentialRecord;
-  const { credentialSubject, issuer, issuanceDate } = credential;
+  const { credentialSubject, issuer, issuanceDate, expirationDate } = credential;
 
   const title = credentialSubject.hasCredential?.name ?? '';
   const description = credentialSubject.hasCredential?.description ?? '';
-  const formattedIssuanceDate = moment(issuanceDate).format(DATE_FORMAT);
   const subjectName = credentialSubject.name;
   const numberOfCredits = credentialSubject.hasCredential?.awardedOnCompletionOf?.numberOfCredits?.value ?? '';
 
+  const issuanceDateFmt = moment(issuanceDate).format(DATE_FORMAT);
+  const expirationDateFmt = moment(expirationDate).format(DATE_FORMAT);
```

Comment:
Does this line work when there is no `expirationDate` in the VC?

Reply:
Oh, good catch! Technically it would because `moment(undefined)` results in the current timestamp, but we should make that conditional just in case. I'll go ahead and update that.